docs/coding-guidelines/updating-ref-source.md

```diff
@@ -3,7 +3,7 @@ This document provides the steps you need to take to update the reference assemb
 ## For most assemblies within libraries
 
 1. Implement the API in the source assembly and [build it](../workflow/building/libraries/README.md#building-individual-libraries). Note that when adding new public types, this might fail with a `TypeMustExist` error. The deadlock can be worked around by disabling the `RunApiCompat` property: `dotnet build /p:RunApiCompat=false`.
-2. Run the following command (from the src directory) `dotnet msbuild /t:GenerateReferenceAssemblySource` to update the reference assembly**.
+2. Run the following command (from the src directory) `dotnet msbuild /t:GenerateReferenceAssemblySource` to update the reference assembly**. When a project cross builds for multiple target frameworks, usually you need to specify the target framework to generate the reference assembly source for; you can do that by specifying it via `/p:BuildTargetFramework` MSBuild parameter.
```

Comment:
Why can't we just infer the `BuildTargetFramework` by default? Pretty much all libraries are (or will be) building for `$(NetCoreAppCurrent)` - see https://github.com/dotnet/runtime/issues/54012

Reply:
We don't infer the BuildTargetFramework whenever we are building an individual project using the command line I.e dotnet build as that was one of the efforts @ViktorHofer drove to look more like regular customer apps where by default all target frameworks are built.

Reply:
But this isn't `dotnet build`. This is the "generate reference source" command. Why not make it as simple as possible?

Reply:
Right. Let me try that.